python/ray/dashboard/dashboard.py

```diff
@@ -176,9 +188,54 @@ def log_dashboard_url(self):
     def run(self):
         self.log_dashboard_url()
         self.node_stats.start()
+        self.raylet_stats.start()
         aiohttp.web.run_app(self.app, host=self.host, port=self.port)
 
 
+class RayletStats(threading.Thread):
```

Comment:
Can we move this below the `NodeStats` class to be consistent with the ordering in the other changes above?